viewer/packages/cad-styling/src/TreeIndexNodeCollection.ts

```diff
@@ -43,14 +47,51 @@ export class TreeIndexNodeCollection extends NodeCollectionBase {
     return this._treeIndices;
   }
 
+  getAreas(): AreaCollection {
+    if (!this._areaCollection) {
+      throw new Error(
+        `The AreaCollection returned by getAreas() for ThreeIndexNodeCollection must be constructed manually using addAreas() and addAreaPoints() or created through initializeAreaCollection()`
+      );
+    }
+
+    return this._areaCollection;
+  }
+
+  initializeAreaCollection(): void {
+    this._areaCollection = new ClusteredAreaCollection();
+  }
```

Comment:
The reason I have this, is to make it completely transparent that areas are not added automatically when adding tree indices to TreeIndexNodeCollection. The user has to make the conscious decision to take responsibility for creating the areas themselves.

Reply:
Is there a better way of making this obvious to the user?

Reply:
I see your point. I still think that `initializeAreaCollection()` is too cumbersome and that an error from `getAreas()` saying that `addAreas` must be called before using it would be sufficient. An exception would be that for an empty `TreeIndexCollection` it probably be best to return an empty area collection. What do you think?